### PR TITLE
fix: add sort_order to v2 search params

### PR DESCRIPTION
**Pull request by nolbuzanis**
Fixes PLhery/node-twitter-api-v2#236

### CODE REVIEW

File: src/types/v2/tweet.v2.types.ts [hunk 1]
```diff
@@ -22,6 +22,8 @@ export interface TweetV2TimelineParams extends Partial<Tweetv2FieldsParams> {
 export interface Tweetv2SearchParams extends TweetV2TimelineParams {
   previous_token?: string;
   query: string;
+  sort_order?: 'recency' | 'relevancy';
+  max_results?: number;
```
Comment (PLhery) [inline src/types/v2/tweet.v2.types.ts:26]: max_results is already in TweetV2TimelineParams, so let's remove it from there

But awesome for the sort_order, thank you 🙏

**Comment by PLhery** [comment]
Related: https://github.com/PLhery/node-twitter-api-v2/issues/237

**Comment by nolbuzanis** [comment]
Hoping to get more involved with this project if you need more help!

**Comment by alkihis** [comment]
Many thanks for your PR! I will take a look in the next few days when I have time for this :smile:

**Comment by PLhery** [comment]
LGTM, merging :) thank you

**Comment by PLhery** [comment]
And thanks a million, any future contribution would be really appreciated, ex on the #237 ☺️

**Comment by nolbuzanis** [comment]
Sounds good!

**Pull request merged**